sdk/iot/README.md

```diff
@@ -1,14 +1,39 @@
-# Azure IoT Clients for Embedded C
+# Embedded C for Azure IoT Clients

```

Comment:
@ericwol-msft This new name pattern is different from how we name clients.  I would have expected
   Azure IoT Clients for Embedded C
ex:
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage

https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/keyvault
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/keyvault
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault